discretize/TreeMesh.py

```diff
@@ -166,11 +166,11 @@ def cellGradStencil(self):
         if getattr(self, '_cellGradStencil', None) is None:
 
             self._cellGradStencil = sp.vstack([
-                self._cellGradxStencil(), self._cellGradyStencil()
+                self._cellGradxStencil, self._cellGradyStencil
```

Comment:
now each component is a property (to be consistent with the other meshes and so that SimPEG can always grab them as properties individually)

Reply:
the caching will always work here because this code will always be executed in Python